diff --git a/node_modules/gatsby-source-wordpress/gatsby-node.js b/node_modules/gatsby-source-wordpress/gatsby-node.js
index 8a2cea6..e5bfd05 100644
--- a/node_modules/gatsby-source-wordpress/gatsby-node.js
+++ b/node_modules/gatsby-source-wordpress/gatsby-node.js
@@ -197,34 +197,6 @@ exports.sourceNodes = async ({
     normalizer: ({
       entities
     }) => normalize.mapEntitiesToMedia(entities)
-  }, // Downloads media files and removes "sizes" data as useless in Gatsby context.
-  {
-    name: `downloadMediaFiles`,
-    normalizer: async ({
-      entities,
-      store,
-      cache,
-      createNode,
-      createNodeId,
-      touchNode,
-      getCache,
-      getNode,
-      auth,
-      reporter,
-      keepMediaSizes
-    }) => await normalize.downloadMediaFiles({
-      entities,
-      store,
-      cache,
-      createNode,
-      createNodeId,
-      touchNode,
-      getCache,
-      getNode,
-      _auth: auth,
-      reporter,
-      keepMediaSizes
-    })
   }, // Creates links between elements and parent element.
   {
     name: `mapElementsToParent`,
